### PR TITLE
example: remote

### PR DESCRIPTION
**Pull request by Peyton-Spencer**
Remote example for: #39

### CODE REVIEW

File: example/remote/main.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	"log/slog"
+	"os"
+	"time"
+
+	_ "github.com/tursodatabase/go-libsql"
+	"github.com/tursodatabase/go-libsql/example/remote/envs"
+)
+
+func main() {
+	setupSlog()
```
Comment (haaawk) [inline example/remote/main.go:15]: Please drop setupSlog. It's an unnecessary distraction.

File: example/remote/main.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	if err := run(); err != nil {
+		slog.Error("error running example", "error", err)
+		os.Exit(1)
+	}
+}
+
+func run() (err error) {
+	err = envs.Load()
```
Comment (haaawk) [inline example/remote/main.go:23]: Please drop envs package fully and just get envs using `os` package.

**Review by haaawk — COMMENTED** [review]
Thanks for your contribution. It looks good but please remove all the files except for example/remote/main.go and make the example as plain as possible.

**Comment by Peyton-Spencer** [comment]
thanks for review @haaawk and will simplify

**Comment by Peyton-Spencer** [comment]
@haaawk simplified to single file!

**Review by haaawk — APPROVED** [review]

**Comment by haaawk** [comment]
Thanks @Peyton-Spencer

**Pull request merged**